### DISCUSSION TITLE
Please add the Option to “Hide” Shortcut’s section In the “Home” Tab.

### DISCUSSION THREAD

**Discussion by Gta211** [community/community#144990] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.185.1 (147102269)
Device: iPhone 14 Pro Max (iOS 18.0.1)
</details>

**Comment by stevepopovich** [comment]
Closing as duplicate of https://github.com/orgs/community/discussions/146060
